articles/lets-learn-about-osi-layer-model/index.md

```diff
@@ -0,0 +1,133 @@
+- Perform troubleshooting, by identifying which network layer is causing an issue and focusing efforts on that layer.
+
+The OSI model helps network device manufacturers and networking software, vendors:
+- Create devices and software that can communicate with products from any other vendor, allowing open interoperability.
+- Define which parts of the network their products should work with.
+- Communicate to users at which network layers their product operates 
+
+### Characteristics of OSI Model:
+
+OSI consists of several layers and each layer performs a particular network function. It's divided into two layers: upper layers and lower layers.
+
+The upper layer of the OSI model mainly deals with the application related issues, and they are implemented only in the software. The application layer is closest to the end-user. Both the end-user and the application layer interact with the software applications. An upper layer refers to the layer just above another layer.
+
+The lower layer of the OSI model deals with data transport issues. The data link layer and the physical layer are implemented in hardware and software. The physical layer is the lowest layer of the OSI model and is closest to the physical medium. The physical layer is mainly responsible for placing the information on the physical medium.
+
+OSI follows a vertical approach. ”If you can understand the OSI model and its layers, you can also then understand which protocols and devices can interoperate with each other when new technologies are developed and explained.
+
+- A layer should only be created where definite levels of abstraction are needed.
+- The function of each layer should be selected as per the internationally standardized protocols.
+- The number of layers should be large so that separate functions should not be put in the same layer. At the same time, it should be small enough so that architecture doesn't become very complicated.
+- In the OSI model, each layer relies on the next lower layer to perform primitive functions. Every level should able to provide services to the next higher layer.
+- Changes made in one layer should not need changes in other lavers.
+ 
+As mention, the model uses layers to help give a visual description of what is going on with a particular networking system. This can help network managers narrow. 
+
+### Functions of the OSI Layers
+There are seven OSI layers. Each layer has different functions. Each layer is assigned a particular task. OSI model divides the whole task into seven smaller and manageable tasks.  Each layer is self-contained so that tasks assigned to each layer can be performed independently. A list of seven layers are given below:
+- Physical Layer
+- Data-Link Layer
+- Network Layer
+- Transport Layer
+- Session Layer
+- Presentation Layer
+- Application Layer
+
+#### Physical layer
+- The main functionality of the physical layer is to transmit the individual bits from one node to another node.
```

Comment:
If you mention it is the lowest layer, the reader might get confused looking at the order given above. Ensure you maintain consistency.